### PR TITLE
Adding code review info #5

### PR DESCRIPTION
**Pull request by melissamcewen**
Adding some info about standards of code review. Feedback welcome.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +70,17 @@ Next we'll workshop Tweets and social media images. Feel free to leave this to u
...
 
 ## Appendix
+### Code Review
+Most Dev.to posts involve code in the form of:
+- Embedded Glitch apps
+- Code snippets in Markdown 
+
+We do not have a strict code review process or standards, but we like to see:
+- well-commented code
+- a README.md in all Glitch apps
```
Comment (AngeloStavrow) [inline README.md:80]: I'd recommend a LICENSE file (probably MIT) be added, too.

**Review by AngeloStavrow — CHANGES_REQUESTED** [review]
If one of the CTAs involves remixing (and even if it doesn't), publicly-distributed code should include a LICENSE file. Otherwise, looks good to me!

File: README.md [hunk 2]
```diff
@@ -70,6 +70,17 @@ Next we'll workshop Tweets and social media images. Feel free to leave this to u
...
+- files named in a way that will make sense I.E. "feedparser.js" rather than "file1.js"
+
```
Comment (smonette) [inline README.md]: Nit: Might be worth linking to the Launch Checklist app, or Lyzi's guidence around Starter apps?

**Review by smonette — APPROVED** [review]
One small suggestion, but overall looks good! 🚀

File: README.md (hunk 1 above)
Comment (smonette) [reply README.md:80]: @AngeloStavrow This is a bigger question for the group...but I wonder if we should go back through all the starters we made for `/@guides` and add licenses?

Comment (AngeloStavrow) [reply README.md:80]: IMHO yes! I think there are a few things to check for on those starters, like making sure they've been added to the `@guides` team and that they have a license file.

**Pull request merged**